treeherder/webapp/graphql/schema.py

```diff
@@ -61,6 +61,16 @@ class Meta:
         model = Repository
 
 
+class OptionCollectionGraph(DjangoObjectType):
+    class Meta:
+        model = OptionCollection
+
+
+class OptionGraph(DjangoObjectType):
```

Comment:
It doesn't need , actually.  These are for the "top level" of your query.  You COULD include it, technically.  But that would only be if you were going to query for just the ``options`` by themselves.  However, for us, that wouldn't be useful.  It would just show you the option ``name`` and ``id``.  

What we care about is the mapping of ``optionCollectionHash`` to ``options``.  So the ``resolve_all_option_collections`` handles that.  It will give you each ``optionCollection`` with the ``options`` that apply to it.